tests/test_realm.py

```diff
@@ -46,3 +48,111 @@ async def test_realm_realmCreated(websocket, context_id, html):
             "context": context_id,
         }
     }, ) == response
+
+
+@pytest.mark.asyncio
+async def test_realm_realmCreated_sandbox(websocket, context_id):
+
+    await subscribe(websocket, "script.realmCreated")
+
+    await send_JSON_command(
+        websocket, {
+            "id": 21,
+            "method": "script.evaluate",
+            "params": {
+                "target": {
+                    "context": context_id,
+                    "sandbox": "__test_sandbox__"
+                },
+                "expression": "2 + 2",
+                "awaitPromise": True
+            }
+        })
+
+    response = await read_JSON_message(websocket),
+
+    # Wait for `script.realmCreated` event.
+    assert ({
+        "method": "script.realmCreated",
+        "params": {
+            "type": "window",
+            "origin": "null",
+            "realm": ANY_STR,
+            "context": context_id,
+            "sandbox": "__test_sandbox__"
+        }
+    }, ) == response
+
+
+@pytest.mark.asyncio
+async def test_realm_realmDestroyed(websocket, context_id):
+
+    await subscribe(websocket, "script.realmDestroyed")
+
+    await send_JSON_command(
+        websocket, {
+            "id": 12,
+            "method": "browsingContext.close",
+            "params": {
+                "context": context_id,
+            }
+        })
+
+    response = await read_JSON_message(websocket),
+
+    # Wait for `script.realmCreated` event.
+    assert ({
+        "method": "script.realmDestroyed",
+        "params": {
+            "realm": ANY_STR,
+        }
+    }, ) == response
+
+
+@pytest.mark.asyncio
+async def test_realm_realmDestroyed_sandbox(websocket, context_id):
+
+    await subscribe(websocket, "script.realmDestroyed")
+
+    await execute_command(
+        websocket, {
+            "id": 21,
+            "method": "script.evaluate",
+            "params": {
+                "target": {
+                    "context": context_id,
+                    "sandbox": "__test_sandbox__"
+                },
+                "expression": "2 + 2",
+                "awaitPromise": True
+            }
+        })
+
+    await send_JSON_command(
+        websocket, {
+            "id": 22,
+            "method": "browsingContext.close",
+            "params": {
+                "context": context_id,
+            }
+        })
+
+    response = await read_JSON_message(websocket),
```

Comment:
extra comma at the end of this line makes the `response` to be a tuple, which forces the assertion to have that weird wrapper 
```
({
    ...
}, ) == response
```